src/OrbitGl/FrameTrack.cpp

```diff
@@ -69,14 +69,12 @@ FrameTrack::FrameTrack(CaptureViewElement* parent, TimeGraph* time_graph,
   collapse_toggle_->SetCollapsed(true);
 }
 
-float FrameTrack::GetHeaderHeight() const { return layout_->GetTrackTabHeight(); }
-
 float FrameTrack::GetHeight() const {
-  return GetHeaderHeight() + GetMaximumBoxHeight() + layout_->GetTrackBottomMargin();
+  return GetHeaderHeight() + GetMaximumBoxHeight() + layout_->GetTrackContentBottomMargin();
 }
 
-float FrameTrack::GetYFromTimer(const TimerInfo& /*timer_info*/) const {
-  return pos_[1] - GetHeaderHeight() - GetMaximumBoxHeight();
+float FrameTrack::GetYFromTimer(const TimerInfo& timer_info) const {
+  return pos_[1] + GetHeaderHeight() + (GetMaximumBoxHeight() - GetDynamicBoxHeight(timer_info));
```

Comment:
I would like to see how it works if FrameTrack is expanded

Reply:
Here's a screenshot: https://screenshot.googleplex.com/39YcrkKqxaVHhX5